OutputMessages.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Displays a message about the type of sequence.
+ */
+public class OutputMessages {
+  /**
+   * Displays the type of sequence.
+   * @param typeOfSequence flag for output of sequence type.
+   */
+  public void outputMessages(boolean typeOfSequence) {

```

Comment:
nope, typeOfSequence supposes enum not boolean. Better argument name isNotDecreasing (or something like that)